solo-project/client/src/components/app/add-event/component.js

```diff
@@ -0,0 +1,128 @@
+/*global google*/
+import React from 'react';
+import PropTypes from 'prop-types';
+import DatePicker from 'react-datepicker';
+import moment from 'moment';
+import 'react-datepicker/dist/react-datepicker.css';
+import Geosuggest from 'react-geosuggest';
+
+import './index.css';
+
+export class component extends React.Component {
+  static propTypes = {
+    getEvents: PropTypes.func,
+    createEvent: PropTypes.func
+  };
+
+  state = {
+    date: moment(),
+    title: '',
+    description: '',
+    location: '',
+    coords: { lat: null, lng: null },
+    ageFrom: '',
+    ageTo: '',
+    price: '',
+    error: null
+  };
+
+  handleChange = date => {
+    this.setState({ date: date });
```

Comment:
You can use shorthand notation `this.setState({ date });` if key and value have the same name.